### PR TITLE
types/Pcx -js

### PR DESCRIPTION
**Pull request by zhuzhuzu1**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `pnpm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:

- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:

- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the `package.json`.

If removing a declaration:

- [ ] If a package was never on Definitely Typed, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

**Comment by zhuzhuzu1** [comment]
types/Pcx -js

File: types/pcx-js/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  _lengthRLE(offset:number):number;
+  getPalette():void;
+  setColorFromPalette(pos:number,index:number):void;
+  decode4bpp():void;
+  decode8bpp(ctx?:any):void;
+
+
+}
+declare namespace PCX{
+  export interface Header{
```
Comment (codershiba) [inline types/pcx-js/index.d.ts:22]:
```suggestion
  interface Header{
```

File: types/pcx-js/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    xmin: number,
+    xmax: number,
+    ymin: number,
+    ymax: number,
+    hdpi: number,
+    vdpi: number,
+    palette: null|Uint8Array,
+    bitplanes: number
+  }
+  export interface DecodeType{
```
Comment (codershiba) [inline types/pcx-js/index.d.ts:34]:
```suggestion
  interface DecodeType {
```

**Comment by DangerBotOSS** [comment]
## Formatting

The following files are not formatted:
1. types/time-ampm/tsconfig.json
1. types/time-ampm/index.d.ts
1. types/time-ampm/package.json
1. types/pcx-js/tsconfig.json
1. types/time-ampm/time-ampm-tests.ts

<details>
<summary>as well as these 3 other files...</summary>
<p>types/pcx-js/package.json, types/pcx-js/index.d.ts, types/pcx-js/pcx-js-tests.ts</p>
</details>


Consider running `pnpm dprint fmt` on these files to make review easier.
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 281ca639bd10595dbae7a5bdbe9a43d45ae14bb3
</p>

**Comment by codershiba** [comment]
Why is `time-ampm` being added here?. There seems to be a separate PR #69148 for it.

**Pull request closed without merging**